docs/installation.md

```diff
@@ -91,7 +91,7 @@ build:
 
 # deployment monitoring with Argo Watcher
 watch:
-  image: ghcr.io/shini4i/argo-watcher-client:v0.0.12
+  image: ghcr.io/shini4i/argo-watcher:v0.0.12
```

Comment:
This is misleading. Version `v0.0.12` did not have a client part in the main image.

Let's replace the version with a placeholder `<version>` or something like this.